telegram/files/location.py

```diff
@@ -32,17 +32,50 @@ class Location(TelegramObject):
     Attributes:
         longitude (:obj:`float`): Longitude as defined by sender.
         latitude (:obj:`float`): Latitude as defined by sender.
+        horizontal_accuracy (:obj:`float`): Optional. The radius of uncertainty for the location,
+            measured in meters.
+        live_period (:obj:`int`): Optional. Time relative to the message sending date, during which
+            the location can be updated, in seconds. For active live locations only.
+        heading (:obj:`int`): Optional. The direction in which user is moving, in degrees.
+            For active live locations only.
+        proximity_alert_radius (:obj:`int`): Optional. Maximum distance for proximity alerts about
+            approaching another chat member, in meters. For sent live locations only.
 
     Args:
         longitude (:obj:`float`): Longitude as defined by sender.
         latitude (:obj:`float`): Latitude as defined by sender.
+        horizontal_accuracy (:obj:`float`, optional): The radius of uncertainty for the location,
+            measured in meters; 0-1500.
+        live_period (:obj:`int`, optional): Time relative to the message sending date, during which
+            the location can be updated, in seconds. For active live locations only.
+        heading (:obj:`int`, optional): The direction in which user is moving, in degrees; 1-360.
+            For active live locations only.
+        proximity_alert_radius (:obj:`int`, optional): Maximum distance for proximity alerts about
+            approaching another chat member, in meters. For sent live locations only.
         **kwargs (:obj:`dict`): Arbitrary keyword arguments.
 
     """
 
-    def __init__(self, longitude: float, latitude: float, **_kwargs: Any):
+    def __init__(
+        self,
+        longitude: float,
+        latitude: float,
+        horizontal_accuracy: float = None,
+        live_period: int = None,
+        heading: int = None,
+        proximity_alert_radius: int = None,
+        **_kwargs: Any,
+    ):
         # Required
         self.longitude = float(longitude)
         self.latitude = float(latitude)
 
+        # Optionals
+        self.horizontal_accuracy = horizontal_accuracy
```

Comment:
why arent you doing a float(variable) if else here as well?

Reply:
my though about `int()` was that it would convert a falsy passed `10.0` to `10`, but I'm not sure if even that is necessary … handling strings is not really necessary imo.

Reply:
I mean we are doing it in 70/71 as well with longitude and latitude

Reply:
ah. yes. will edit